scss/detail.scss

```diff
@@ -0,0 +1,34 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+.titulo-libro{
+  color: black;
```

Comment:
Try to use the hexacodes for colors instead of the color names. Also use variables for the colors so in case one color changes you only have to change de color of that variable and not every rule that used the color directly.
Tip: It's advisable to create a separate file for the variables and import it.

Reply:
Yeah, a "variables.scss" will prove handy!
But it should only contain sass variables (don't forget the $), in your case
```scss
$black: #000000;
```

Those variables can be used across the other files (variables.scss should be imported of course)